types/got/got-tests.ts

```diff
@@ -6,7 +6,6 @@ import * as fs from 'fs';
 import * as http from 'http';
 import * as https from 'https';
 import * as url from 'url';
-import QuickLRU = require('quick-lru');
```

Comment:
Is it possible import this module from `npm`?

Reply:
It's not compatible in the sense that it uses TS 3.0+ features. This would require to move this definiton to TS 3.0 just to support a test case. I think it's not worth it.